File: 4week/dajung/javaScript/0105_점의위치구하기.js
//프로그래머스 level 0
//https://school.programmers.co.kr/learn/courses/30/lessons/120841#

//정수 배열 dot
function solution(dot) {
    if (dot[0] > 0){
        if(dot[1] > 0)
            return 1
        return 4
    } else {
        if (dot[1] > 0)
            return 2
        return 3
    }
}

File: 5week/jihwan/python/0105_점의위치구하기.py
# //프로그래머스 level 0
# //https://school.programmers.co.kr/learn/courses/30/lessons/120841#

def solution(dot):
  
        if(dot[0] > 0 and dot[1] > 0) :
            return 1
        elif(dot[0] < 0 and dot[1] > 0) :
            return 2
        elif(dot[0] > 0 and dot[1] < 0) :
            return 4         
        elif(dot[0] < 0 and dot[1] < 0) :
            return 3
               
        